app/src/main/res/layout/activity_user_item_level2.xml

```diff
@@ -250,23 +312,25 @@
                     android:layout_weight="1"
                     android:gravity="center"
                     android:textAlignment="center"
-                    android:text="Fill a Collins glass with ice"
+                    android:text="@string/example_step"
                     android:textAppearance="@style/TableContentText" />
             </TableRow>
 
-            <TextView
```

Comment:
when errors outside the TableLayout, you need to add margin_start, otherwise it's too right no? maybe i'm wrong

Reply:
I will check that, I think you right but I'm not sure .
when you move this line up with the table layout ,when the error message appears its show before the example line . So it's need to be in this order

Reply:
> I will check that, I think you right but I'm not sure .
> when you move this line up with the table layout ,when the error message appears its show before the example line . So it's need to be in this order

you are right about the order, i missed it! I just open a PR to fix the margin :)